.eslintrc.js

```diff
@@ -25,7 +25,10 @@ module.exports = {
         "eslint:recommended",
         "plugin:@typescript-eslint/eslint-recommended",
         "plugin:@typescript-eslint/recommended"
-      ]
+      ],
+      rules: {
+        "@typescript-eslint/no-var-requires": 0
```

Comment:
What's this rule?

Reply:
this lets us use `require` for imports. it's needed for compatibility with our existing JS code.